### PR TITLE
Add tests for Stratis man page files

### PR DESCRIPTION
**Pull request by bgurney-rh**
Closes: #116

### CODE REVIEW

File: stratisd_cert.py [hunk 1]
```diff
@@ -334,6 +334,17 @@ def _test_permissions(self, dbus_method, args, permissions, *, kwargs=None):
 
         self.assertEqual(_permissions_flag, permissions)
 
+    def test_access_stratisd_man_page(self):
```
Comment (mulkieran) [inline stratisd_cert.py:337]: You should go ahead and make a separate class for this test. Call the class something like StratisdManPageCertify and make sure it inherits from StratisCertify.

File: stratis_cli_cert.py [hunk 2]
```diff
@@ -204,6 +204,12 @@ def unittest_command(
         else:
             self.assertNotEqual(stdout, "")
 
+    def test_access_stratis_man_page(self):
```
Comment (mulkieran) [inline stratis_cli_cert.py:207]: You should do a similar thing for as for stratisd_cert.py here. However, the class heirarchy right now is simpler than in stratisd_cert.py. It's not ready to have a new class added. It'll be necessary to make at least two new classes, similar to the classes that are already defined in stratisd_cert.py.

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: stratis_cli_cert.py (hunk 2 above)
Comment (bgurney-rh) [reply stratis_cli_cert.py]: OK, should be all set.

**Review by mulkieran — APPROVED** [review]
This looks good.

**Pull request merged**